package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "promise": "^8.1.0",
+    "uuid": "^8.3.2"
+  }
+}
```

Comment:
We are not using it, right? So why keep them?

Reply:
Sure thing.